client/src/RelatedItemsComparison/components/ProductCard/ProductCard.jsx

```diff
@@ -58,13 +65,13 @@ export default function ProductCard({
     axios.get(`/products/${id}/styles`)
   );
 
-  // const getRatings = () => (
-  //   axios({
-  //     url: '/reviews/meta',
-  //     method: 'GET',
-  //     params: { product_id: id },
-  //   })
-  // );
+  const getRatings = () => (
```

Comment:
Not sure if you're getting calling this for multiple products or just the main one. If the main, the data is available at (state.ratingsReviews.meta)